test/p2p/sanity.spec.ts

```diff
@@ -37,14 +37,14 @@ describe('P2P Sanity Tests', () => {
   let nodeTwo: Xud;
 
   before(async () => {
-    nodeOneConfig = createConfig(1, 9001);
-    nodeTwoConfig = createConfig(2, 9002);
+    const config = new Config().load();
```

Comment:
I've been thinking, since the config loading process is now synchronous and expected to stay that way, should we just call `load()` from within the constructor and skip having to make this second call? It feels like it would be a lot cleaner that way.

Reply:
Indeed, but i'm usually not in favor of introducing complicated fetching procedures in the constructor, even though they are synchronous. It might also be good to be able to get the config object, without overriding it with the TOML config.